tests/test_fetch.py

```diff
@@ -63,5 +104,5 @@ def test_download_file_in_chunks(safe_data, tmp_path):
 
 
 def test_download_file_none():
-    with pytest.raises(requests.exceptions.InvalidURL):
+    with pytest.raises(requests.exceptions.MissingSchema):
```

Comment:
this looks like it will still satisfy the error checking at https://github.com/ASFHyP3/hyp3-lib/blob/develop/hyp3lib/get_orb.py#L131 , I think that's the only place we're leveraging this exception :+1: